dotcom-rendering/src/layouts/ImmersiveLayout.tsx

```diff
@@ -33,9 +33,9 @@ import { Island } from '../components/Island';
 import { LabsHeader } from '../components/LabsHeader';
 import { MainMedia } from '../components/MainMedia';
 import { Masthead } from '../components/Masthead/Masthead';
+import { minHeaderHeightPx } from '../components/Masthead/Titlepiece/constants';
 import { MostViewedFooterData } from '../components/MostViewedFooterData.importable';
 import { MostViewedFooterLayout } from '../components/MostViewedFooterLayout';
-import { minNavHeightPx } from '../components/Nav/Nav';
```

Comment:
Looks like we can then remove the associated const in the nav component and just set the `minNavHeight` to 48px?

<img width="321" alt="image" src="https://github.com/user-attachments/assets/0ccdd3a7-b5bd-4566-9821-b3e8c5cbcbca">